test/dsl_test.go

```diff
@@ -55,21 +58,51 @@ func test(t testing.TB, actions ...optionOp) {
 	}
 }
 
+func parallel(actions ...optionOp) optionOp {
+	return func(o *opt) {
+		wg := new(sync.WaitGroup)
```

Comment:
You don't copy it, though, you're using a closure.  But you're right, I guess in theory someone could change the code to copy it.  If you want to use a pointer, `wg := sync.WaitGroup{}` is more standard in our code base.